ang/civicase/ActivityPanel.js

```diff
@@ -60,6 +62,37 @@
         element.find('.crm-submit-buttons a.edit').addClass('btn btn-primary');
       }
 
+      /**
+       * Set height for activity panel
+       */
+      function setPanelHeight () {
+        $timeout(function () {
+          var $filter = $('.civicase__activity-filter');
+          var $tabs = $('.civicase__dashboard').length > 0 ? $('.civicase__dashboard__tab-container ul.nav') : $('.civicase__case-body_tab');
+          var $toolbarDrawer = $('#toolbar');
+          var topOffset = $toolbarDrawer.height() + $tabs.height() + $filter.outerHeight();
```

Comment:
the same calculation is happening also in `ActivityFeed.js`, could we have something reusable to avoid duplication?